src/components/TodoForm.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React, { useState, useEffect, useRef } from 'react';
+
+function TodoForm(props) {
+  const [input, setInput] = useState(props.edit ? props.edit.value : '');
+
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    inputRef.current.focus();
+  });
```

Comment:
I suggest removing this part of the code, hence this part is used in an input element, so this makes the input element focused but this is never triggered by an onClick method or a similar event.